rozy/src/main.rs

```diff
@@ -256,33 +270,14 @@ fn update(path_to_ozy: &std::path::PathBuf, url: &Option<String>) -> Result<()>
         perms.set_mode(0o755);
         std::fs::set_permissions(&dest_path, perms)?;
 
-        let mut command = exec::Command::new(&dest_path);
-        command.arg("update");
-
-        // We use exec to replace this process with the child entirely. The only time exec() returns is
-        // if there was an error.
-        let err = command.exec();
+        let err = exec::execvp(&dest_path, std::env::args().collect::<Vec<String>>());
         return Err(anyhow!(
             "Failed to execute process {} ({})",
             dest_path.display(),
             err
         ));
     }
 
-    let new_apps =
-        std::collections::HashSet::<app::App>::from_iter(get_apps(&new_config)?.into_iter());
-    let old_apps =
-        std::collections::HashSet::<app::App>::from_iter(get_apps(&old_config)?.into_iter());
-
-    let to_delete = old_apps.difference(&new_apps);
-
-    for app in to_delete {
```

Comment:
This is a fundamentally incorrect thing to do since per-directory ozy configs can add/change apps and cause them to fight.